logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/DownloadAssetMessageMetadata.kt

```diff
@@ -0,0 +1,8 @@
+package com.wire.kalium.logic.feature.asset
+
+class DownloadAssetMessageMetadata(
+    val assetId: String,
```

Comment:
Should we call this `assetKey` in order to avoid confusion with recently created object `AssetId` ?

Reply:
Agree! will make the change :D